app/models/invitation_to_organization.rb

```diff
@@ -0,0 +1,50 @@
+class InvitationToOrganization < Invitation
+  belongs_to :member, inverse_of: :created_invitations, foreign_key: :invited_by_id
```

Comment:
@shir I write one method in this model, another code I move from old model Invitation. I don't refactor them because you and Alexander asked me to leave the code as he is, because of the size of the code review. If you approve, I refactor this code.